### PR TITLE
Better E-mail for new user

### PR DESCRIPTION
**Pull request by gkrishnan724**
[MIFOSX-2799]
When a new user is registered in community app, the email was complicated to understand.
Changed it to a more understandable way.

### CODE REVIEW

File: fineract-provider/src/main/java/org/apache/fineract/infrastructure/core/service/GmailBackedPlatformEmailService.java [hunk 1]
```diff
@@ -57,18 +57,20 @@ public void sendToUserAccount(final EmailDetail emailDetail, final String unenco
         	}
         	email.setFrom(authuser, authuserName);
 
-            final StringBuilder subjectBuilder = new StringBuilder().append("Fineract Prototype Demo: ").append(emailDetail.getContactName())
-                    .append(" user account creation.");
+            final StringBuilder subjectBuilder = new StringBuilder().append("Welcome ").append(emailDetail.getContactName())
```
Comment (nazeer1100126) [inline fineract-provider/src/main/java/org/apache/fineract/infrastructure/core/service/GmailBackedPlatformEmailService.java:60]: The subject line should be "New User Information" or 'Welcome to [Organization] Name'

Comment (gkrishnan724) [inline fineract-provider/src/main/java/org/apache/fineract/infrastructure/core/service/GmailBackedPlatformEmailService.java]: Okay, I'll do the changes :)

Comment (gkrishnan724) [reply fineract-provider/src/main/java/org/apache/fineract/infrastructure/core/service/GmailBackedPlatformEmailService.java:60]: Okay, I'll do the changes :)

**Comment by gkrishnan724** [comment]
@nazeer1100126  I performed the changes  using git commit --amend is it alright?

**Comment by nazeer1100126** [comment]
No, You need to send new PR

**Comment by gkrishnan724** [comment]
Okay I will

**Pull request closed without merging**